File: iOS/XCode Projects/CodeWithChris/DiceRoll/DiceRoll/ViewController.m
//
//  ViewController.m
//  DiceRoll
//
//  Created by Christopher Ching on 2013-09-19.
//  Copyright (c) 2013 CodeWithChris. All rights reserved.
//

#import "ViewController.h"
#import "DiceDataController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)rollClicked:(id)sender {
    
    DiceDataController *diceDataController = [[DiceDataController alloc] init];
    
    int firstNumber = [diceDataController getDieNumber];
    int secondNumber = [diceDataController getDieNumber];
    
    [self.firstDieView showDieNumber:firstNumber];
    [self.secondDieView showDieNumber:secondNumber];
    
    self.sumLabel.text = [NSString stringWithFormat:@"The sum is %d", firstNumber + secondNumber];
}
@end

File: iOS/XCode 6 Projects/SwiftExample1/SwiftExample1/ViewController.swift
//
//  ViewController.swift
//  SwiftExample1
//
//  Created by Brian Sterner on 6/5/14.
//  Copyright (c) 2014 DHAP Digital Inc. All rights reserved.
//

import UIKit
//import SwiftClasses

class ViewController: UIViewController {
                            
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        println("Hello World")
        
        let car = Car()
        println("Car: \(car.description())")
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

